static/src/javascripts/projects/common/utils/ajax-promise.js

```diff
@@ -13,6 +15,11 @@ define([
             })
             .fail(function (request, text, err) {
                 var error = err ? err : new Error(text);
+
+                var statusText = (err && err.statusText) || '';
+                var statusCode = (err && err.status) || '';
+                raven.captureException('Error retrieving data (' + text + ') (Status: ' + statusCode + ') (StatusText: ' + statusText + ')');
```

Comment:
Forgot to mention, I have checked for usages of `fail` throughout the project off `ajax-promise` and didn't really find any, for the case of double `raven` reporting.

The only thing I don't like about this is that we loose the `weather` tag in `raven`.